### DISCUSSION TITLE
2.3

### DISCUSSION THREAD

**Discussion by akechi-haruka** [akechi-haruka/ChronoCoreFixes#2] [Announcements]
Now with Tick Sync! Allows players with different framerates to play together. Only works on GMG.

<hr /><em>This discussion was created from the release <a href='https://github.com/akechi-haruka/ChronoCoreFixes/releases/tag/2.3'>2.3</a>.</em>

**Comment by namusung** [comment]


**Reply by akechi-haruka** [reply]
Hi!

You can state the problems you have with chrono fixes (or something else) simply here
